Book-A-Desk.Api/OfficesHttpHandler.fs

```diff
@@ -77,3 +76,16 @@ module rec OfficesHttpHandler =
                     context.SetStatusCode(500)
                     return! text ("Internal Error: " + e) next context
         }
+        
+    let private handleGetByDateFromEventStore eventStore getOffices date officeId = asyncResult {
+        let (ReservationId aggregateId) = ReservationAggregate.Id
+        let! bookingEvents = eventStore.GetEvents aggregateId
```

Comment:
Similar to the previous handler, we've changed it so the events are fetched in the API and then passed to the Domain.